generative/maisi/scripts/diff_model_infer.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import random
+import torch
+import torch.distributed as dist
+import nibabel as nib
+import numpy as np
+
+from datetime import datetime, timedelta
+from tqdm import tqdm
+
+from monai.inferers import sliding_window_inference
+from monai.utils import set_determinism
+
+from utils import define_instance, load_autoencoder_ckpt
+
+
+class ReconModel(torch.nn.Module):
```

Comment:
There's a ReconModel under `generative/maisi/scripts/sample.py`. The class seems the same as the one here.

Reply:
Hi @mingxin-zheng, I have updated the notebook. Please proceed. Thanks.